OrbitClientServices/CrashManager.cpp

```diff
@@ -9,6 +9,8 @@
 
 namespace {
 
+constexpr uint64_t kTimeoutMilliseconds = 10;
```

Comment:
I would increase it a little bit - to 100ms probably. I do not know how grpc behaves if deadline is in the past.

Reply:
Done, thanks. Probably if deadline in the past, it just do nothing: I set it to 1ms and OrbitService didn't crash (and no log records too).